modules/instrument_list/php/instrument_list.class.inc

```diff
@@ -296,7 +296,10 @@ class Instrument_List extends \NDB_Menu_Filter
      */
     function getFeedbackPanel($candID, $sessionID): string
```

Comment:
The signature (sessionID) disagrees with the current change.
This function is called somewhere with the wrong argument. Specifying the types in the function definition will help to clarify the confusion.